aptos-move/framework/move-stdlib/src/natives/bcs.rs

```diff
@@ -0,0 +1,105 @@
+// Copyright © Aptos Foundation
+
+// Copyright (c) The Diem Core Contributors
+// Copyright (c) The Move Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::natives::helpers::make_module_natives;
+use move_binary_format::errors::PartialVMResult;
+use move_core_types::{
+    gas_algebra::{InternalGas, InternalGasPerByte, NumBytes},
+    vm_status::sub_status::NFE_BCS_SERIALIZATION_FAILURE,
+};
+use move_vm_runtime::native_functions::{NativeContext, NativeFunction};
+use move_vm_types::{
+    loaded_data::runtime_types::Type,
+    natives::function::NativeResult,
+    pop_arg,
+    values::{values_impl::Reference, Value},
+};
+use smallvec::smallvec;
+use std::{collections::VecDeque, sync::Arc};
+/***************************************************************************************************
```

Comment:
Can you add comments to note that this and other native function files are copied from <from_path>?